### PR TITLE
feature/flightinfo

### PR DESCRIPTION
**Pull request by Diren-T**
Enter flights details in form

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,3 +1,66 @@
...
+  const [trips, setTrips] = useState([]);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    setTrips([...trips, { departure, destination, passengerCount }]);
+    setDeparture("");
+    setDestination("");
+    setPassengerCount(1);
+  };
+
```
Comment (doemser) [inline pages/index.js:14]: In this case I think it makes little sense to use controlled inputs. Rather use formData here. A good reason to use controlled inputs is if you want to prefill the form, which is not the case here.

File: pages/index.js [hunk 2]
```diff
@@ -1,3 +1,66 @@
...
+  return (
+    <section>
+      <form onSubmit={handleSubmit}>
+        <label htmlFor="">from</label>
+        <input
+          id=""
+          type="text"
+          maxLength="3"
+          placeholder="IATA code"
```
Comment (doemser) [inline pages/index.js:27]: Placeholders tend to be an a11y issue. Either the contrast is too low, or they are mistaken to be prefilled textContent. Consider not setting a placeholder.

File: pages/index.js [hunk 3]
```diff
@@ -1,3 +1,66 @@
...
+          value={departure}
+          onChange={(event) => setDeparture(event.target.value)}
+          required
+        />
+        <br />
```
Comment (doemser) [inline pages/index.js:32]: using <br/> to add a "new line" is kinda dirty. Can you think of another way to do that?

**Review by doemser — CHANGES_REQUESTED** [review]

**Review by TariqAlQadi — COMMENTED** [review]
Looks good!
still wouldnt use className

File: pages/index.js [hunk 4]
```diff
@@ -1,3 +1,87 @@
...
+        destination: formData.get("destination"),
+        passengerCount: formData.get("passengerCount"),
+      },
+    ]);
+    event.target.reset();
+  };
+
+  return (
+    <section>
+      <form className="form" onSubmit={handleSubmit}>
```
[the remark this answers is not in the document]
Comment (Diren-T) [reply pages/index.js:42]: thanks, i will change this :)

**Pull request merged**